src/Examples.Console/Examples.Console.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.3.2" />
```

Comment:
huh? did we not need a reference to OTel before? (nb, obv)

Reply:
This ensures that the example app uses the latest version even if the one that would be pulled in transitively via the ProjectReference below would be the lowest supported one (1.2.0).